<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendRequsetsItem</class>
 <widget class="QWidget" name="FriendRequsetsItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="LabelPlus" name="lbl_user_avatar">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>14</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_user_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_accept">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>44</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>接受√</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reject">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>44</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>拒绝×</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelPlus</class>
   <extends>QLabel</extends>
   <header>LabelPlus.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
